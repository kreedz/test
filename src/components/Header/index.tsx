import React from 'react';


export default () => <p className="text-uppercase">Foo to the bar!!!</p>;
