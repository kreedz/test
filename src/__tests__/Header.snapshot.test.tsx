import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

test('Header component renders the header correctly', () => {
    const rendered = renderer.create(
        <Header />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
});
