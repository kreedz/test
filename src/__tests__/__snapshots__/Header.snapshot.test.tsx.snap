// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the header correctly 1`] = `
<p
  className="text-uppercase"
>
  Foo to the bar!!!
</p>
`;
